.git-blame-ignore-revs

```diff
@@ -0,0 +1,11 @@
+# This file lists commits that should be ignored by the `git blame` command.
+# You can configure your checkout to use it by default by running:
+#
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+#
+# Or pass it to the blame command manually:
+#
+#   git blame --ignore-revs-file .git-blame-ignore-revs ...
+#
+# Note that these features require Git 2.23 (released August 2019).
+355c65a31f107bca5d340f00a234f7909fb51fae
```

Comment:
Is this correct.
'fade to black' commit is 86ef1de2a30fe3f5a59f371b9f737390781a1090

Reply:
Oh woops